roles/job/templates/job_definition.yml.j2

```diff
@@ -20,6 +20,8 @@ spec:
               value: "{{ tower_config_secret['resources'][0]['data']['host'] | b64decode }}"
             - name: TOWER_JOB_TEMPLATE_NAME
               value: "{{ job_template_name }}"
+            - name: TOWER_WORKFLOW_TEMPLATE_NAME
+              value: "{{ workflow_template_name }}"
```

Comment:
Now that we have the ability to pass either a JT name or a WFJT name, we will need to add a jinja2 conditional here to check if the variable is defined.  Otherwise, we will get:

```
    Message:               The task includes an option with an undefined variable. The error was: 'workflow_template_name' is undefined

The error appears to be in '/opt/ansible/roles/job/tasks/main.yml': line 102, column 3, but may
be elsewhere in the file depending on the exact syntax problem.
```

Reply:
Something like:
```
{% if job_template_name is defined %}
            - name: TOWER_JOB_TEMPLATE_NAME
              value: "{{ job_template_name }}"
{% endif %}
{% if workflow_template_name is defined %}
            - name: TOWER_WORKFLOW_TEMPLATE_NAME
              value: "{{ workflow_template_name }}"
{% endif %}
```

Reply:
Done